compiler/src/iree/compiler/Codegen/Common/test/reconcile_translation_info.mlir

```diff
@@ -144,6 +181,8 @@ hal.executable private @llvm_func_attrs {
     }
   }
 }
+
+// TODO: This test makes no sense if fn1 and fn2 are called together.
```

Comment:
What is the action item for the TODO? If the test makes no sense, do we delete it?

(Honestly, I don't really follow why it makes no sense. Would you mind elaborating a bit?)

Reply:
I can clarify in the comment. It's because `waves-per-eu` is a kernel level attribute, so we can't call two functions with differing values from the same kernel entry point. This doesn't apply to all `llvm_func_attrs` though so it's a fundamental problem I don't know how to fix. Best I can think of is we don't bother verifying and hope the backend gives us a warning.

Reply:
Thanks! I'm not familiar with the `llvm_func_attrs`, so I did not pay attention to the contents.

Could it contain non-kernel level attribute? When I browsed the codebase, there is other spec like `"denormal-fp-math-f32" = "preserve-sign"` in a matmul spec. If so, maybe we can polish the symbol of `hal.executable` a bit to reflect that it has a kernel level attribute.

https://github.com/iree-org/iree/blob/809061cdfc369a32d9990651a41aae9708fe373b/tests/external/iree-test-suites/test_suite_files/attention_and_matmul_spec_punet_mi300.mlir#L60

Reply:
I ended up just changing the test a little bit so that it made more sense and dropped the TODO.